policyengine_us/variables/gov/states/va/tax/income/subtractions/va_federal_state_employees_subtraction.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class va_federal_state_employees_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia federal state employees subtraction"
+    defined_for = StateCode.VA
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/"
+    )
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.va.tax.income.subtractions.disability_income
+
+        total_income = person("total_income", period)
```

Comment:
Revise tests as well. Total income would include non-employment income like investment income.
```suggestion
        total_income = person("employment_income", period)
```